wkcuber/api/Layer.py

```diff
@@ -293,6 +276,85 @@ def downsample(
     def setup_mag(self, mag: Union[str, Mag]) -> None:
         pass
 
+    def upsample(
+        self,
+        from_mag: Mag,
+        min_mag: Optional[Mag],
+        compress: bool,
+        anisotropic: Optional[bool] = None,
+        scale: Optional[Tuple[float, float, float]] = None,
```

Comment:
Why is `scale` an argument? Shouldn't that be part of the dataset properties?

Reply:
Yes your right. I have updated it.
The code was initially copied from `downsampling` which still has the parameter 'scale' (optionally) for backwards compatibility